src/main/java/org/myrobotlab/service/data/Position.java

```diff
@@ -0,0 +1,43 @@
+package org.myrobotlab.service.data;
+
+public class Position {
```

Comment:
why , yet another point class?  what about roll,pitch & yaw?

Reply:
I could reuse Point3f I suppose

Reply:
There's also `org.myrobotlab.kinematics.Point`, and for roll, pitch, and yaw there's `org.myrobotlab.service.data.Orientation`

Really, all of these should be consolidated into one at some point, with a `Point` specifying only coordinates, an `Orientation` specifying only rotational values, and `Position` containing both a Point marking its origin and an Orientation

Reply:
added https://github.com/MyRobotLab/myrobotlab/issues/1363